### ISSUE TITLE
Augment analytics data with auto-incrementing workspace number

### ISSUE THREAD

**Issue by loujaybee** [gitpod-io/gitpod#6959]
There is no easy way to see usage of the 1st, 2nd, 3rd (etc) workspaces launched by a user for the purposes of analytics (e.g. visualize the events/features that a user is interacting with on each subsequent visit to a workspace). This would enable us to build a retention graph similar to the one in the insights report [1]. It would useful if this value could be added as property on existing analytics events, to avoid post-processing and keep our analytics in Mixpanel. 

Also see [this internal thread](https://gitpod.slack.com/archives/C01KPEPKX0A/p1637919827149000)

**Comment by LEDlastchance** [comment]
I understand I apologize for any trouble. My account was hacked by someone in your outfit. Someone who stole my real business name to make smut over seas with it while I worked my butt to the bone and took care of her helped her get her kids back. That's the truth. It's already being looked into by Grit, Google, Facebook,and also my federaly insured AND PROTECTED invoice program and app. And I really am at a Teacher's UNION you do know what a union is don't you. Also a FEDERALLY insured and protected organization. I haven't had access to my account in 2 weeks. Oh this is being looked at also by Motorola because I proved and uncovered that I have had Java and other codes and software programed on my phone since last year. My privacy,my business and business name and my name has been drug they crap and I have been body shamed to I don't know how many millions of people by some who I welcomed into my with nothing but a backpack full of clothes. And no kids but I got those back too. For three years. As she's cheating on me posting it on line and then coming home and telling me she loves me. That's all on my mother's grave. I built this business with my two hands for over 20 years. The same business and business name that she hacked and started a account on LinkedIn in the email jadebenarik20@gmail.com. I have that down loaded too it's my business I'll send you whatever paperwork you want.Ivw bleed for this you think I won't give up paper. I don't understand why we haven't been together for 4 months I haven't even talked to her in two months. Why why still mess with me why still messing with my name my business name. This Grit account is mine the U Tube account that just got flags ran for being inappropriate is mine started on this device in my name all documented on WI Fi that is protected and insuraned by the United states government. The Admin for this web is a customer of mine. I don't want no trouble for you or for me. I want to be left alone by online bullyies that have been stalking and hurassing me for a year now and stabbing me in the back for three. For what because someone got caught up in a lie because I found out that she was a slut whore. I'm the bad guy no freinds I am not. I mean this rediculess and whoever reads and I don't care who because it's the goddam truth and I'll take a lie detector whatever I'll go to prison I'll die to fight for and protect what is mine. You wouldn't then your a coward. Thats the truth on my mother's grave and if we can be cool then let's be cool if you want to play around well let's play. And that's whats up your ball don't be stupid.
